libs/designsystem/src/lib/components/modal/services/modal.helper.spec.ts

```diff
@@ -220,7 +234,7 @@ describe('ModalHelper', () => {
 
           describe(`drawer`, () => {
             beforeEach(async () => {
-              await openDrawer('Drawer On Presenting Element');
+              await openDrawer('Drawer On Presenting Element', undefined, 'medium', true);
```

Comment:
Probably better to move the test regarding `interactWithBackground`  into it's own `describe` block and not modify the regular drawer tests

Reply:
Changed in latest commit